melos.yaml

```diff
@@ -12,35 +12,181 @@ ignore:
 scripts:
   setup_tuneup: >
     flutter pub global activate tuneup
-
   copy_dummy_config: >
     melos exec --scope="*example*,sample_app" -- \
       cp -n "\$MELOS_ROOT_PATH"/.circleci/dummy_amplifyconfiguration.dart lib/amplifyconfiguration.dart | true
+  build:examples:release:ios: >
+    melos exec -c 1 --scope="*example*" --ignore="*_android*" --fail-fast -- \
+      flutter build ios --no-codesign
+
+  build:examples:release:android: >
+    melos exec -c 1 --scope="*example*" --ignore="*_ios*" --fail-fast -- \
+      flutter build apk --verbose
+
+  test:unit:flutter:
```

Comment:
For this one, I was thinking more like:

```
  test:unit:flutter:
    run: melos exec -c 1 -- "$MELOS_ROOT_PATH/build-support/unit_test_for_platform.sh --flutter"
    select-package:
      dir-exists:
        - test
 ```

but the CI would call with --no-select `melos run test:unit:flutter --no-select`